src/effects/VST3/VST3Effect.cpp

```diff
@@ -481,45 +473,23 @@ RegistryPaths VST3Effect::GetFactoryPresets() const
    return mFactoryPresets;
 }
 
-bool VST3Effect::LoadFactoryPreset(int id, EffectSettings &) const
+bool VST3Effect::LoadFactoryPreset(int id, EffectSettings& settings) const
 {
    if(id >= 0 && id < mFactoryPresets.size())
    {
       auto filename = wxFileName(GetFactoryPresetsPath(mEffectClassInfo), mFactoryPresets[id] + ".vstpreset");
       // To do: externalize state so const_cast isn't needed
-      return const_cast<VST3Effect*>(this)->LoadPreset(filename.GetFullPath());
+      const bool res = const_cast<VST3Effect*>(this)->LoadPreset(filename.GetFullPath());
+      FetchSettings(GetSettings(settings));
+      return res;
    }
-   return true;
-}
-
-bool VST3Effect::LoadFactoryDefaults(EffectSettings &) const
```

Comment:
In the comment of the commit that removes this function, add more explanation.  Quote the rest of my bullet point.